src/features/user/components/ChangePasswordForm/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,69 @@
+import { useAppDispatch } from "@/app/hooks"
+import FormikTextField from "@/components/FormikTextField/FormikTextField"
+import changePasswordSchema, {
+  ChangePasswordFormValues,
+} from "@/libs/yup/schemas/changePassword"
+import Grid from "@mui/material/Grid/Grid"
+import { Form, Formik } from "formik"
+import { changeUserPassword } from "../../userThunk"
+import SubmitFormButton from "../../../../components/SubmitFormButton/SubmitFormButton"
+import ResetFormButton from "../../../../components/ResetFormButton/ResetFormButton"
+import { useTranslation } from "react-i18next"
+
+const ChangePasswordForm = () => {
+  const dispatch = useAppDispatch()
+  const { t } = useTranslation(["forms"])
+  const changePasswordInitialValues: ChangePasswordFormValues = {
+    newPassword: "",
+    confirmNewPassword: "",
+  }
+  return (
+    <Formik
```

Comment:
Instead of creating your own dirty and resetForm logic, just add enableReinitialize prop to the formik settings